test/extended/testdata/test-cli-debug.yaml

```diff
@@ -0,0 +1,69 @@
+kind: List
+apiVersion: v1
+items:
+- kind: ImageStream
+  apiVersion: v1
+  metadata:
+    name: busybox
+  spec:
+    tags:
+    - name: latest
+      from:
+        name: busybox
+        kind: DockerImage
+
+- kind: DeploymentConfig
+  apiVersion: v1
+  metadata:
+    name: busybox1
+  spec:
+    replicas: 0
+    selector:
+      deploymentconfig: busybox1
+    template:
+      metadata:
+        labels:
+          deploymentconfig: busybox1
+      spec:
+        containers:
+        - name: busybox
```

Comment:
As for the correlation between docker image names and imagestream names, no i don't think i'd expect that although i don't have a total understanding of it.  but case in point, here's my imagestream named "mystream" with a ref to the docker repo centos/ruby-22-centos7.. as you can see it has both a "dockerimagerepository" reference to the internal registry, but also a dockerimagereference to the actual dockerhub repo name:

```
  status:
    dockerImageRepository: 172.30.189.29:5000/default/mystream
    tags:
    - items:
      - created: 2017-01-10T15:06:51Z
        dockerImageReference: centos/ruby-22-centos7@sha256:aace2b3ef701be868ef3a1feff7f1d766caa9e80026ba55dbf2d2e9c8f37bf7e
        generation: 2
        image: sha256:aace2b3ef701be868ef3a1feff7f1d766caa9e80026ba55dbf2d2e9c8f37bf7e
      tag: latest
```

and if we look at the build that ultimately picks up the image via an imagechangetrigger we see that the imagestream reference is to "mystream" but the imageid is centos/ruby-22-centos7:

```
   - imageChangeBuild:
        fromRef:
          kind: ImageStreamTag
          name: mystream:latest
        imageID: centos/ruby-22-centos7@sha256:aace2b3ef701be868ef3a1feff7f1d766caa9e80026ba55dbf2d2e9c8f37bf7e
```